src/ws-header/ws-header.js

```diff
@@ -333,9 +343,9 @@ export class WSHeader extends Component {
             <ul className="main-sub-menu" key={`sub-menu${index}`} ref={element => { this.subMenus[index] = element; }}>
               {parent.children.map((child, childIndex) =>
                 <li key={`sub-link-${index}-${childIndex}`} className={(child.isCurrent) ? 'is-current' : null}>
-                  <a href={child.href} onClick={event => { if (child.onClick) child.onClick(event); }}>
-                    {child.label}
-                  </a>
+                {
```

Comment:
Pls configure your formatter to keep the formatting like thise `{this.renderLink(link)}`

Reply:
👍